modules/binding-infra-node-workloads-using-taints-tolerations.adoc

```diff
@@ -1,20 +1,48 @@
 // Module included in the following assemblies:
 //
 // * machine_management/creating-infrastructure-machinesets.adoc
+// * post_installation_configuration/cluster-tasks.adoc
 
 [id="binding-infra-node-workloads-using-taints-tolerations_{context}"]
 = Binding infrastructure node workloads using taints and tolerations
 
 If you have an infra node that has the `infra` and `worker` roles assigned, you must configure the node so that user workloads are not assigned to it.
 
+[IMPORTANT]
+====
+It is recommended that you preserve the dual `infra,worker` label that is created for infra nodes and use taints and tolerations to manage nodes that user workloads are scheduled on. If you remove the `worker` label from the node, you must create a custom pool to manage it. A node with a label other than `master` or `worker` is not recognized by the MCO without a custom pool. Maintaining the `worker` label allows the node to be managed by the default worker machine config pool, if no custom pools that select the custom label exists. The `infra` label communicates to the cluster that it does not count toward the total number of subscriptions.
+====
+
 .Prerequisites
 
 * Configure additional `MachineSet` objects in your {product-title} cluster.
 
 .Procedure
 
-. Use the following command to add a taint to the infra node to prevent scheduling user workloads on it:
+. Add a taint to the infra node to prevent scheduling user workloads on it:
+
+.. Determine if the node has the taint:
++
+[source,terminal]
+----
+$ oc describe nodes <node_name>
+----
++
+.Sample output
+[source,yaml]
+----
+oc describe node ci-ln-iyhx092-f76d1-nvdfm-worker-b-wln2l
+Name:               ci-ln-iyhx092-f76d1-nvdfm-worker-b-wln2l
+Roles:              worker
+ ...
+Taints:             node-role.kubernetes.io/infra:NoSchedule
+ ...
+----
 +
+This example shows that the node has a taint. You can proceed with adding a toleration to your pod in the next step.
+
+.. If you have not configured a taint on your machine set, use the following command to add a taint to the infra node to prevent scheduling user workloads on it:
```

Comment:
I noticed that you omitted "the following command" on line 22.  You could also omit here: "...machine set, add a taint..."

Up to you.